content/docs/command-reference/metrics/index.md

```diff
@@ -71,22 +71,15 @@ to compare and pick the best performing experiment.
 
 ### Default metric files
 
-`dvc metrics` subcommands use all metric files that are specified in `dvc.yaml`
-by default. There's no need to specify metric file names to see these metrics.
-Metric files can be added to `dvc.yaml` with the `--metrics` (`-m`) or
-`--metrics-no-cache` (`-M`) options of `dvc run`, or manually to the `metrics`
-section of a stage in `dvc.yaml`:
+`dvc metrics` subcommands by default use the metric files specified in
```

Comment:
why do we need the whole subsection for this? I mean `### Default metric files`?

Reply:
Right I also had that thought when reviewing this at first. It's kind of nice because of the long YAML code block though. But I can move all this up to the main description before `### Supported file formats` if you prefer, and maybe just shorten the code block.

Reply:
yep, let's try that!

Reply:
Done in 439f8e01